tests/fxa.test.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const got = require("got");
+
+const FXA = require("../lib/fxa");
+
+
+jest.mock("got");
+
+
+test("revokeOAuthToken calls oauth destroy with fxa_refresh_token", async () => {
+  // from db/seeds/test_subscribers.js
+  const token = "4a4792b89434153f1a6262fbd6a4510c00834ff842585fc4f4d972da158f0fc1";
```

Comment:
Same comment as above. Should we duplicate these UUIDs vs trying to export them from db/seeds/test_subscribers.js (or somewhere else shared). Or possibly not worth it. 🤷‍♀️